pages/facts.html

```diff
@@ -1,12 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <base href="../">
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Otters Fun Facts</title>
+  <link rel="stylesheet" href="styles/style.css">
+  <link rel="stylesheet" href="styles/facts.css">
 </head>
-<body>
-  
+<body  class="container">
+  <header>
+    <h1> Fun Facts About Otters</h1>
+    <nav>
+      <ul>
+        <li class="title"><a href="index.html">Home Page</a></li>
+        <li class="title"><a href="pages/gallery.html">Gallery</a></li>
+
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section id="fun_facts">
+      <ol>
+        <li>There are thirteen different species of otters around the world.</li>
+        <li>River otters weigh on average 10-30 pounds.</li>
+        <li>Sea otters weigh on average 45-90 pounds.</li>
+        <li>The sea otter is the largest member of the weasel family, and it is the smallest marine mammal in North America.</li>
+        <li>Sea otters have the thickest fur of any animal.</li>
+        <li>Otters use tools such as rocks to break open hard-shelled prey.</li>
+        <li>A group of resting otters is called a raft.</li>
+        <li>River otters, however, can hold their breath for up to 8 minutes.</li>
+        <li>Sea otters have been known to stay submerged for more than 5 minutes at a time.</li>
+        <li>An otter’s lung capacity is 2.5 times greater than that of similar-sized land mammals.</li>
+      </ol>
+      <figure>
+        <img src="images/GroupofSeaOtters.jpg" width="400" height="300" alt="Group of Otters"><figcaption>Group of Otters</figcaption>
+      </figure>
+    </section>
+
+  </main>
+
 </body>
+<a class="source" href="https://www.doi.gov/blog/12-facts-about-otters-sea-otter-awareness-week#:~:text=Their%20fur%20contains%20between%20600%2C000,grooming%20and%20conditioning%20their%20fur." target="_blank">Source For Facts</a>
```

Comment:
I think this a tag could be in the footer because the [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer) say "A <footer> typically contains information about the author of the section, copyright data or links to related documents." and I think "Source for Facts" can be considered links to related documents.